tide_landing_page.install

```diff
@@ -1189,3 +1190,78 @@ function tide_landing_page_update_8031() {
 
   $config->save();
 }
+
+/**
+ * Add field display headings for table of contents.
+ */
+function tide_landing_page_update_8032() {
```

Comment:
We seem to be doing this a lot these days... i.e. add a field storage, add it to the view display, add it to JSONAPI... can it be split out to a helper function of its own?

Reply:
hmm yes, I think we should create something like tide_single_config_import as we are doing a lot of config imports.